docs/proposals/templates.md

```diff
@@ -0,0 +1,550 @@
+<!-- BEGIN MUNGE: UNVERSIONED_WARNING -->
+
+<!-- BEGIN STRIP_FOR_RELEASE -->
+
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+
+<h2>PLEASE NOTE: This document applies to the HEAD of the source tree</h2>
+
+If you are using a released version of Kubernetes, you should
+refer to the docs that go with that version.
+
+<strong>
+The latest release of this document can be found
+[here](http://releases.k8s.io/release-1.1/docs/proposals/templates.md).
+
+Documentation for other releases can be found at
+[releases.k8s.io](http://releases.k8s.io).
+</strong>
+--
+
+<!-- END STRIP_FOR_RELEASE -->
+
+<!-- END MUNGE: UNVERSIONED_WARNING -->
+
+# Templates+Parameterization: Repeatedly instantiating user-customized application topologies.
+
+## Motivation
+
+Addresses https://github.com/kubernetes/kubernetes/issues/11492
+
+Kubernetes gives developers a platform on which to run images and many configuration objects to control those images, but 
+constructing a cohesive application made up of images and configuration objects is currently difficult.  Applications 
```

Comment:
@bgrant0607 IMO, this is the correct overall goal for templating. This may be a different overall goal than fine-grain scoping or creating multiple resources for cron/scheduled job cases.